app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/address/multiple_disposals_summary.scala.html

```diff
@@ -39,14 +40,16 @@
     answers: CompleteExamplePropertyDetailsAnswers,
     shouldAskIfPostcodeExists: Boolean,
     isATrust: Boolean,
-    representativeType: Option[RepresentativeType]
+    representativeType: Option[RepresentativeType],
+    dateOfDeath: Option[DateOfDeath]
 )(implicit request: RequestWithSessionData[_], messages:Messages)
 
 @isAgent = @{ request.userType.contains(UserType.Agent) }
 @userKey = @{
     representativeType match {
         case Some(PersonalRepresentative) => ".personalRep"
-        case Some(PersonalRepresentativeInPeriodOfAdmin) => ".personalRep"
+        case Some(PersonalRepresentativeInPeriodOfAdmin) if (!isAgent) => ".personalRepInPeriodOfAdmin"
+        case Some(PersonalRepresentativeInPeriodOfAdmin) if (isAgent) => ".personalRepInPeriodOfAdmin.agent"
```

Comment:
it doesn't look like the disposal price title has a case for an agent of period of admin

Reply:
you can either make it have one or have a special user key for the the disposal price title only